pages.pt_BR/common/renice.md

```diff
@@ -0,0 +1,17 @@
+# renice
+
+> Altera a prioridade/agradabilidade de agendamento de um ou mais processos em execução.
+> Os valores de agradabilidade variam de -20 (mais favorável ao processo) a 19 (menos favorável ao processo).
+> Mais informações: <https://manned.org/renice>.
+
+- Altera a prioridade de um processo em execução:
+
+`renice -n {{niceness_value}} -p {{pid}}`
+
+- Altera a prioridade de todos os processos pertencentes a um usuário:
+
+`renice -n {{niceness_value}} -u {{user}}`
```

Comment:
Please translate the placeholders, like `niceness_value`, `user`, etc., for get this easier for non-English speakers.
```suggestion
`renice -n {{niceness_value}} -u {{nome_do_usuario}}`
```

Reply:
Altractions performed.
Thanks.